platform/security/cas/security-cas-client/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -35,13 +35,13 @@
         </AD>
 
         <AD name="CAS Server Logout URL:" id="casServerLogoutUrl" required="true" type="String"
-            default="https://localhost:8443/cas/logout"
+            default="${org.codice.ddf.external.protocol}${org.codice.ddf.external.hostname}:8443/cas/logout"
```

Comment:
✏️ Just an observation - the variable substitution in this metatype could cause some confusion since not all fields support it. Not the fault of this PR, though, it was like that before.